stdlib/sys.jou

```diff
@@ -0,0 +1,12 @@
+# System functions
+import "stdlib/str.jou"
+declare getenv(varname: byte*) -> byte*
+
+def is_windows() -> bool:
+    return getenv("OS") != NULL and strstr(getenv("OS"), "Windows") != NULL
```

Comment:
As I said in #383 I would prefer checking these at compile-time rather than runtime.

Reply:
So, what should we do?

Reply:
Compile-time if statements to check the platform, in a separate PR.

Reply:
Fine, is that meaning I should remove sys.jou in this or?

Reply:
Yes.